From: Boaz Harrosh <bharrosh@panasas.com>
Date: Mon, 15 Aug 2011 11:52:36 -0700
Subject: [PATCH] osd: Kconfig remove wrong FIXME
Git-commit: a8f8c45082cb78489c5e6050c372ddac73ae439e
References: FATE#313625
Patch-Mainline: v3.2

The OSD protocol calls for all kind of security levels that use
CRYPTO_HMAC and SH1, but the current code only supports NO_SEC,
which does not use any of these.

Remove a wrong FIXME that calls for them. Thanks Maxin for
reporting on this.

Reported-by: "Maxin B. John" <maxin.john@gmail.com>
Signed-off-by: Boaz Harrosh <bharrosh@panasas.com>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 drivers/scsi/osd/Kconfig |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/drivers/scsi/osd/Kconfig b/drivers/scsi/osd/Kconfig
index 861b5ce..a070351 100644
--- a/drivers/scsi/osd/Kconfig
+++ b/drivers/scsi/osd/Kconfig
@@ -11,10 +11,6 @@
 # it under the terms of the GNU General Public version 2 License as
 # published by the Free Software Foundation
 #
-# FIXME: SCSI_OSD_INITIATOR should select CONFIG (HMAC) SHA1 somehow.
-#        How is it done properly?
-#
-
 config SCSI_OSD_INITIATOR
 	tristate "OSD-Initiator library"
 	depends on SCSI
-- 
1.7.4.2

